website_oca_integrator/models/sponsorship.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Surekha Technologies (https://www.surekhatech.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class SponsorshipLine(models.Model):
+    _name = "sponsorship.line"
+
+    date_from = fields.Date(string='Join Date', required=True)
+    date_end = fields.Date(string='End Date', required=True)
+    sponsorship_id = fields.Many2one(comodel_name='product.product',
```

Comment:
I don't think the product is relevant here and can be removed from here and the view.

Reply:
product is added for current sponsorship badge in integrator listing.

Reply:
@st-aprajapati Can you point me where.  I need to review my notes on this, but I think we were going to use a different image than the product image.

Reply:
![integrator-list](https://user-images.githubusercontent.com/40430352/44444663-8ade5100-a5fb-11e8-8cf2-67aaa5cefd6e.png)
@gdgellatly : Here is the screenshot for your reference.